<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite  djdn" duration-ms="111303" started-at="2020-05-19T13:47:47Z" finished-at="2020-05-19T13:49:38Z">
    <groups>
    </groups>
    <test name="Test Actitime" duration-ms="111303" started-at="2020-05-19T13:47:47Z" finished-at="2020-05-19T13:49:38Z">
      <class name="com.qa.TestCases.LoginTest">
        <test-method status="PASS" signature="startExtendReport()[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="startExtendReport" is-config="true" duration-ms="1106" started-at="2020-05-19T13:47:47Z" finished-at="2020-05-19T13:47:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startExtendReport -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="setup" is-config="true" duration-ms="47395" started-at="2020-05-19T13:47:51Z" finished-at="2020-05-19T13:48:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.TestCases.LoginTest.verifyLoginTest(java.lang.String,java.lang.String) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyLoginTest(java.lang.String, java.lang.String)[pri:1, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="verifyLoginTest" duration-ms="8633" started-at="2020-05-19T13:48:38Z" data-provider="getLogineCredensials" finished-at="2020-05-19T13:48:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[manager]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed to retrieve attibute post stale element exception and 2 retries.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed to retrieve attibute post stale element exception and 2 retries.
at com.qa.pages.BasePge.click(BasePge.java:40)
at com.qa.pages.LoginPage.dologine(LoginPage.java:33)
at com.qa.TestCases.LoginTest.verifyLoginTest(LoginTest.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginTest -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="getResult" is-config="true" duration-ms="3606" started-at="2020-05-19T13:48:47Z" finished-at="2020-05-19T13:48:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginTest status=FAILURE method=LoginTest.verifyLoginTest(java.lang.String, java.lang.String)[pri:1, instance:com.qa.TestCases.LoginTest@6b09bb57] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="setup" is-config="true" duration-ms="21775" started-at="2020-05-19T13:48:50Z" finished-at="2020-05-19T13:49:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.TestCases.LoginTest.verifyCheckBox() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="verifyCheckBox()[pri:2, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="verifyCheckBox" duration-ms="1302" started-at="2020-05-19T13:49:12Z" finished-at="2020-05-19T13:49:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyCheckBox -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="getResult" is-config="true" duration-ms="1300" started-at="2020-05-19T13:49:13Z" finished-at="2020-05-19T13:49:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyCheckBox status=SUCCESS method=LoginTest.verifyCheckBox()[pri:2, instance:com.qa.TestCases.LoginTest@6b09bb57] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="setup" is-config="true" duration-ms="14150" started-at="2020-05-19T13:49:15Z" finished-at="2020-05-19T13:49:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.qa.TestCases.LoginTest.verifyErrorMesg() throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyErrorMesg()[pri:3, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="verifyErrorMesg" duration-ms="7176" started-at="2020-05-19T13:49:29Z" finished-at="2020-05-19T13:49:36Z">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Failed to retrieve attibute post stale element exception and 2 retries.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Failed to retrieve attibute post stale element exception and 2 retries.
at com.qa.pages.BasePge.click(BasePge.java:40)
at com.qa.pages.LoginPage.errormsg(LoginPage.java:45)
at com.qa.TestCases.LoginTest.verifyErrorMesg(LoginTest.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyErrorMesg -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="getResult" is-config="true" duration-ms="967" started-at="2020-05-19T13:49:36Z" finished-at="2020-05-19T13:49:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyErrorMesg status=FAILURE method=LoginTest.verifyErrorMesg()[pri:3, instance:com.qa.TestCases.LoginTest@6b09bb57] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="flushReport()[pri:0, instance:com.qa.TestCases.LoginTest@6b09bb57]" name="flushReport" is-config="true" duration-ms="1400" started-at="2020-05-19T13:49:37Z" finished-at="2020-05-19T13:49:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushReport -->
      </class> <!-- com.qa.TestCases.LoginTest -->
    </test> <!-- Test Actitime -->
  </suite> <!-- Suite  djdn -->
</testng-results>
